products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp

```diff
@@ -122,7 +123,7 @@ namespace app::home_screen
     {
         stateController->resetStateMachine();
         getView()->setTimeFormat(timeModel->getTimeFormat());
-        getView()->setTime(timeModel->getCurrentTime());
+        getView()->setTime(23 * 60 * 60 + 12 * 60);
```

Comment:
Consider adding constexpr for explicit information.

Reply:
oh well, that is not a part of this PR, I commited old local change. Removing!